### PR TITLE
Make tickers great again

### PR DESCRIPTION
**Pull request by ccpandhare**
Made tickers work with the CLI:

Before:
Screenshot 2022-04-03 at 06 41 50
After:
Screenshot 2022-04-03 at 08 15 14

### CODE REVIEW

**Review by AshutoshSancheti — COMMENTED** [review]
1. We will add an option of picking up the login credentials either from cmd line or from JSON in the future. For now, json file works.
2. I have just one observation with regards to function return type in the lib/fetcher.py.

File: lib/fetcher.py [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
 
-def fetch(session: [requests.sessions.Session], limit: int) -> [pd.DataFrame]:
+import requests
+import pandas as pd
+import re
+from time import sleep
+from bs4 import BeautifulSoup
+from typing import List
+
+def fetch(session: requests.Session, limit: int) -> List[pd.DataFrame]:
```
Comment (AshutoshSancheti) [inline lib/fetcher.py]: What is the purpose of setting the function return type to List[pd.DataFrame]? The output type is a pd.DataFrame with 3 columns for now.

Comment (ccpandhare) [reply lib/fetcher.py]: Yeah that's a mistake - let me fix that

Comment (ccpandhare) [reply lib/fetcher.py]: Done! @AshutoshSancheti

**Comment by AshutoshSancheti** [comment]
Clean of fetching the stock tickers done. Changes merged

**Comment by AshutoshSancheti** [comment]
Sorry forgot to merge before closing comments

**Comment by AshutoshSancheti** [comment]
Merging complete

**Pull request merged**